SPHINXsys/src/shared/particle_dynamics/solid_dynamics/thin_structure_dynamics.cpp

```diff
@@ -127,17 +127,21 @@ namespace SPH
 		//=================================================================================================//
 		ShellStressRelaxationFirstHalf::
 			ShellStressRelaxationFirstHalf(BaseBodyRelationInner &inner_relation,
-										   int number_of_gaussian_points)
+				int number_of_gaussian_points, bool hourglass_control)
```

Comment:
@mrezavandVirtonomy we need a way to test the stress calculation esp. because of the hourglass

Reply:
yes, it can included in the testing strategies of shell structures

Reply:
Yes.